doc/users/next_whats_new/widget_button_styling.rst

```diff
@@ -0,0 +1,32 @@
+Custom styling of Button widgets
+--------------------------------
+
+Additional custom styling of button widget may be achieved via the
+*label_props* and *radio_props* arguments to `.RadioButtons`; and the
+*label_props*, *frame_props*, and *check_props* arguments to `.CheckButtons`.
+
+.. plot::
+
+   from matplotlib.widgets import CheckButtons, RadioButtons
+
+   fig, ax = plt.subplots(1, 2, figsize=(5, 1), width_ratios=[1, 2])
+   default = RadioButtons(ax[0], ['Apples', 'Oranges'])
+   ax[0].set_title('Default')
+   styled = RadioButtons(ax[1], ['Apples', 'Oranges'],
+                         label_props={'color': ['red', 'orange'],
+                                      'fontsize': [20, 24]},
+                         radio_props={'edgecolor': ['red', 'orange'],
+                                      'facecolor': ['darkred', 'darkorange']})
+   ax[1].set_title('Stylized')
+
+   fig, ax = plt.subplots(1, 2, figsize=(5, 1), width_ratios=[1, 2])
+   default = CheckButtons(ax[0], ['Apples', 'Oranges'],
+                          actives=[True, True])
+   ax[0].set_title('Default')
+   styled = CheckButtons(ax[1], ['Apples', 'Oranges'],
+                         actives=[True, True],
+                         label_props={'color': ['red', 'orange'],
+                                      'fontsize': [20, 24]},
+                         frame_props={'color': ['red', 'orange']},
+                         check_props={'color': ['darkred', 'darkorange']})
```

Comment:
The frame_props and check_props don't seem to be working?
![image](https://user-images.githubusercontent.com/6197628/213555041-94a2c7a3-f18c-41bd-8613-8fdb7ea886a0.png)

Reply:
Sigh, I'm always tripped up by how the colour is translated by `scatter`, which may or may not use `facecolor`/`edgecolor` depending on marker style . What I'm going to do is stop supporting changing the marker style, so we can be explicit about which of `color`/`facecolor`/`edgecolor` are processed. If users later request changing marker style, we can investigate making this more flexible then.